pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-standalone/src/main/java/org/apache/pinot/plugin/ingestion/batch/standalone/SegmentGenerationJobRunner.java

```diff
@@ -238,7 +238,9 @@ private void submitSegmentGenTask(File localTempDir, URI inputFileURI, int seqId
     FileUtils.forceMkdir(localOutputTempDir);
 
     //copy input path to local
-    File localInputDataFile = new File(localInputTempDir, new File(inputFileURI.getPath()).getName());
+    File localInputDataFile = createLocalInputDateFile(inputFileURI, localInputTempDir);
+    System.out.println("localInputDataFile = " + localInputDataFile);
```

Comment:
Avoid system output (is this debugging code)

Reply:
Yeh sorry, have removed.